docs/settings.md

```diff
@@ -44,3 +44,50 @@
 * Response:
     - Status code:
         + `403 Forbidden` When the user is not an admin
+
+## App configuration
+
+**Note:** All app configs are stored as `string` in the `oc_appconfig` database table. Arrays and objects are therefor JSON encoded, integers are casted to string and booleans are replaced with `0/1` or `no/yes` strings.
+
+When available the dedicated UI or OCC command option should be used to configure the setting rather then directly manipulating the database.
+
+Option legend:
+
+* 🖌️ - UI option in the admin settings available
+* 💻 - Dedicated OCC command available
+
+| Key                                  | Internal type                                                    | Default    | Option | Valid values                                                                                                                                                                     |
+|--------------------------------------|------------------------------------------------------------------|------------|--------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `allowed_groups`                     | string[]                                                         | `[]`       | 🖌️    | List of group ids that are allowed to use Talk                                                                                                                                   |
+| `sip_bridge_groups`                  | string[]                                                         | `[]`       | 🖌️    | List of group ids that are allowed to enable SIP dial-in in a conversation                                                                                                       |
+| `start_conversations`                | string[]                                                         | `[]`       | 🖌️    | List of group ids that are allowed to create conversations                                                                                                                       |
+| `hosted-signaling-server-account`    | array                                                            | `{}`       | 🖌️    | Account information of the hosted signaling server                                                                                                                               |
```

Comment:
Default value in https://github.com/nextcloud/spreed/blob/3d66523497e21a0dfde04af7ce1748e02519d4be/lib/Settings/Admin/AdminSettings.php#L451 is `[]`, default value in https://github.com/nextcloud/spreed/blob/a22a546401cfe5546afea968ce74028a2f5eb878/lib/BackgroundJob/CheckHostedSignalingServer.php#L72 is `{}`, is that expected?

Reply:
doesn't matter since it's the same in PHP. Would go with `{}` never the less to decrease confusion when the array is actually filled:
https://3v4l.org/jVOiR